features/step_definitions/service_steps.rb

```diff
@@ -62,5 +62,5 @@
 end
 
 Given /^the service acts as product$/ do
```

Comment:
As a first step ok, but for later on, this name has nothing to do with what it actually does anymore 😂